### DISCUSSION TITLE
付费购买专业版之后没有任何变化且无法退款

### DISCUSSION THREAD

**Discussion by jasonxu74** [getlantern/lantern-forum-cn#572] [Q&A]
今天通过支付宝支付购买了1年的专业版，完成后，没有任何确认邮件，没有任何邀请码（我知道其实自己不需要25位邀请码，因为是官网直接购买的），也无法登录。
下载了最新版的macos的客户端，没有登录地址，仅尝试在授权登录专业版位置，进入通过email登录，输入我在支付宝付款时留的email地址，点“提交”就没有任何反应，尝试多次，下载客户端，重新安装……都没有任何解决可能。
请尽快协助使用，如果不行，请协助退款，着急使用，信任蓝灯才购买的（早在2018年就购买蓝灯，到2020年后停止使用至今了）
我的邮件地址是 jasonxu74@qq.com
![Uploading 截屏 2024-06-14 10.52.44.jpeg…]()

**Comment by jasonxu74** [comment]
![Uploading 截屏 2024-06-14 10.52.44.jpeg…]()

**Comment by jasonxu74** [comment]
![Uploading 2024-06-14_11-37-05.png…]()

**Comment by MaxMKMax** [comment]
@jasonxu74 您好. 您放心, 我会帮您提交问题. 下周给您答复, 谢谢您的耐心等待.

**Reply by jasonxu74** [reply]
什么时候解决呀？这是什么情况？？？？？

**Reply by skytony** [reply]
@Mike777Khoo 大侠，还有我，同样的问题，我的邮箱 79700072@qq.com

**Reply by MaxMKMax** [reply]
@skytony 您好.  如果您也遇到同样的问题, 我帮您处理. 请您提供我支付单据和账号邮箱.
谢谢您支持蓝灯. 我帮您提交问题. 请您过几天查看这个帖子或邮箱. 我们会给您发激活码.

**Comment by jasonxu74** [comment]
什么时候解决？？？？？什么时候解决？？？？？什么时候解决？？？？？什么时候解决？？？？？

**Comment by jasonxu74** [comment]
请客服尽快说说，这什么意思呀？发自我的 iPhone在 2024年6月18日，19:49，skytony ***@***.***> 写道：﻿
@Mike777Khoo 大侠，还有我，同样的问题，我的邮箱 ***@***.***

**Comment by jasonxu74** [comment]
我的什么时候可以用呀发自我的 iPhone在 2024年6月18日，22:28，Mike K ***@***.***> 写道：﻿
@skytony 您好.  如果您也遇到同样的问题, 我帮您处理. 请您提供我支付单据和账号邮箱.
谢谢您支持蓝灯. 我帮您提交问题. 请您过几天查看这个帖子或邮箱. 我们会给您发激活码.

**Reply by MaxMKMax** [reply]
@jasonxu74 
您好. 来了, 我把您的问题放到紧急处理了. 10 分钟后我会和技术团队开会. 请您放心, 我会帮您解决问题.

**Comment by MaxMKMax** [comment]
@jasonxu74 
您把支付单据发到我的邮件来. 这里Github被屏蔽了. 我看不到.
mike@getlantern.org

**Reply by jasonxu74** [reply]
已经发给你邮箱了，还没解决

**Reply by jasonxu74** [reply]
什么时候解决呀？？？？
